### DISCUSSION TITLE
Release a new version and bump the PHP version requirements to >= 7.2 after that

### DISCUSSION THREAD

**Discussion by oliverklee** [MyIntervals/PHP-CSS-Parser#407] [General]
According to https://wordpress.org/about/stats/, PHP version usage for WordPress sites now is < 5% for PHP < 7.2. Hence, according to https://core.trac.wordpress.org/ticket/57345, WordPress is going to raise its minimum required PHP version to 7.2. soon.

I propose we release a new version of this package and then raise the PHP version requirement to 7.2.

@sabberworm @westonruter What do you think?

**Comment by westonruter** [comment]
Yes, I am fine with that.

**Reply by sabberworm** [reply]
Me too

**Reply by oliverklee** [reply]
@sabberworm Will you take care of doing the release? Can I help in any way?

**Comment by oliverklee** [comment]
@sabberworm I've also seen that version 8.4.0 has a tag, but no GitHub release (with change notes) yet. Should I create one?

**Comment by oliverklee** [comment]
We [have released 8.5.0](https://github.com/MyIntervals/PHP-CSS-Parser/releases/tag/8.5.0) now and are working on 9.0.0. Closing.
